composer.json

```diff
@@ -19,10 +19,10 @@
         "doctrine/orm": "^2.5.3",
         "doctrine/annotations": "^1.2",
         "doctrine/phpcr-bundle": "^1.3.5",
-        "doctrine/doctrine-bundle": "~1.0",
+        "doctrine/doctrine-bundle": "~1.6",
         "doctrine/doctrine-cache-bundle": "~1.1",
         "friendsofsymfony/http-cache": "~1.4",
-        "jms/serializer-bundle": "^1.1",
+        "jms/serializer-bundle": "^1.4",
```

Comment:
Required to serialize the `ArrayCollection` of doctrine correctly, in case an element has been deleted.